commands/operator-sdk/cmd/build.go

```diff
@@ -172,7 +172,7 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 
 	log.Infof("Building Docker image %s", baseImageName)
```

Comment:
```suggestion
	log.Infof("Building container image %s", baseImageName)
```

Reply:
I think this can be ignored after rebase: now that #1311 is merged, it's "OCI image".

There are likely other things that need to be changed now that that's merged, such as adding podman to the list of `imageBuilder` options